### PR TITLE
Add links to source

### PR DESCRIPTION
**Pull request by Devinsuit**
https://github.com/d3/d3/issues/2936

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,65 +71,65 @@ The list of supported [named colors](http://www.w3.org/TR/SVG/types.html#ColorKe
 
 Note: this function may also be used with `instanceof` to test if an object is a color instance. The same is true of color subclasses, allowing you to test whether a color is in a particular color space.
 
-<a name="color_opacity" href="#color_opacity">#</a> *color*.<b>opacity</b>
+<a name="color_opacity" href="#color_opacity">#</a> *color*.<b>opacity</b> [<>]( "Source")
```
Comment (mbostock) [inline README.md]: Broken link.

File: README.md [hunk 2]
```diff
@@ -71,65 +71,65 @@ The list of supported [named colors](http://www.w3.org/TR/SVG/types.html#ColorKe
...
+<a name="color_toString" href="#color_toString">#</a> *color*.<b>toString</b>() [<>](https://github.com/d3/d3-color/blob/master/src/color.js#L172 "Source")
 
 Returns a string representing this color according to the [CSS Object Model specification](https://drafts.csswg.org/cssom/#serialize-a-css-component-value), such as `rgb(247, 234, 186)`. If this color is not displayable, a suitable displayable color is returned instead. For example, RGB channel values greater than 255 are clamped to 255.
 
-<a name="rgb" href="#rgb">#</a> d3.<b>rgb</b>(<i>r</i>, <i>g</i>, <i>b</i>[, <i>opacity</i>])<br>
-<a href="#rgb">#</a> d3.<b>rgb</b>(<i>specifier</i>)<br>
-<a href="#rgb">#</a> d3.<b>rgb</b>(<i>color</i>)<br>
+<a name="rgb" href="#rgb">#</a> d3.<b>rgb</b>(<i>r</i>, <i>g</i>, <i>b</i>[, <i>opacity</i>]) [<>]( "Source")<br>
+<a href="#rgb">#</a> d3.<b>rgb</b>(<i>specifier</i>) [<>]( "Source")<br>
+<a href="#rgb">#</a> d3.<b>rgb</b>(<i>color</i>) [<>]( "Source")<br>
```
Comment (mbostock) [inline README.md]: Broken links.

**Comment by mbostock** [comment]
Some missing links here. Let me know when I should look again.

**Comment by Devinsuit** [comment]
What sources should i add to broken links? I have no idea.

**Comment by mbostock** [comment]
Well, for *color*.opacity I’d probably remove the source link, since it’s a field rather than a method and thus by definition has no implementation. (You could I suppose link to the assignment of *rbg*.opacity, but that doesn’t make sense for colors generally.)

For d3.rgb I’d like to [color.js:209](https://github.com/d3/d3-color/blob/master/src/color.js#L209).

For d3.hsl, [color.js:281](https://github.com/d3/d3-color/blob/master/src/color.js#L281).

For d3.lab, [lab.js:30](https://github.com/d3/d3-color/blob/master/src/lab.js#L30). I don’t think you want separate source links for each overloaded variant since they necessarily all point to the same place.

For d3.hcl, [lab.js:87](https://github.com/d3/d3-color/blob/master/src/lab.js#L87).

**Comment by Devinsuit** [comment]
Thanks!

**Pull request merged**